mitodl/mitxpro
# refactor: local seed command to support platforms

#### Pre-Flight checklist


#### What are the relevant tickets?
None

#### What's this PR do?
- Enhances the functionality of the seed_data command to populate the platforms from both json or default otherwise.

#### How should this be manually tested?
Just run `./manage.py seed_data` and you should see Courses and programs with relevant platforms from seed JSON. When there is no platform in Seed JSON, you would see a default one added with the name `xPRO`
